models/Branch.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const BranchSchema = new Schema(
+  {
+    businessId: { type: Schema.Types.ObjectId, ref: "Business", required: true },
+    googlePlaceId: String,
+    additionalInfo: Map,
+    visible: { type: Boolean, default: true },
```

Comment:
Since I think that the convention of naming boolean variables with `is` prefix is handy, let's do that here as well.
Let's rename all the `visible` fields to `isVisible`.

Reply:
Actually, since the default is true here, let's change it to `isHidden` and make it default to `false`. It makes more sense to have a boolean field default to false.

Reply:
(Change it everywhere)